### PR TITLE
Create null1s.json

### PR DESCRIPTION
**Pull request by null1s**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://discodedev.github.io/
https://null1s.is-a.dev/

**Will I be able to edit my website once the domain is active?**

### CODE REVIEW

File: domains/null1s.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "null1s website domain",
+    "repo": "https://github.com/disCodeDev/disCodeDev.github.io",
+    "owner": {
+        "username": "null1s",
+        "email": "null1sdev@gmail.com",
+    },
+    "record": {
+        "CNAME": "disCodeDev.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/null1s.json:9]: Please add content to your website and make it English.

File: domains/null1s.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "null1s website domain",
+    "repo": "https://github.com/disCodeDev/disCodeDev.github.io",
+    "owner": {
+        "username": "null1s",
```
Comment (WilliamDavidHarrison) [inline domains/null1s.json:5]:
```suggestion
        "username": "disCodeDev",
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/null1s.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "null1s website domain",
+    "repo": "https://github.com/disCodeDev/disCodeDev.github.io",
+    "owner": {
+        "username": "disCodeDev",
+        "email": "null1sdev@gmail.com",
+    },
+    "record": {
+        "CNAME": "disCodeDev.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/null1s.json:9]: 404?

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add more content to your website

**Pull request closed without merging**